packages/admin-ui/src/text-input/text-input.tsx

```diff
@@ -1,6 +1,6 @@
 import type { ReactNode, ComponentPropsWithoutRef, Ref } from 'react'
-import React, { forwardRef } from 'react'
-import { useId } from '@vtex/admin-ui-hooks'
+import React, { useRef, forwardRef } from 'react'
+import { useId, useForkRef } from '@vtex/admin-ui-hooks'
```

Comment:
Curious on the naming here `useForkRef`.

Reply:
It is similar to [this one](https://react-hooks.org/docs/useForkRef)

Reply:
The `outer ref` **forks** the `inner ref` => Which means that the `inner ref` forwards some functionality to the `outer ref`.